lib/push/push-to-space.js

```diff
@@ -120,41 +97,36 @@ export default function ({
         result = result
           .then(partial(creation.createEntities,
             {space: space, type: 'Webhook'}, sourceContent.webhooks, destinationContent.webhooks))
-          .delay(prePublishDelay)
-
+        const draftsAssets = sourceContent.assets.filter(({original: asset}) =>
+        (asset.isDraft && asset.isDraft()))
+        const publishedAssets = sourceContent.assets.filter(
+        ({original: asset}) => !(asset.isDraft && asset.isDraft()))
```

Comment:
Isn't there a `asset.isPublished()` method? This will also return `true` if the asset doesn't have a `isDraft` property.